src/styles/utils/borderStyle.ts

```diff
@@ -197,7 +197,9 @@ const borderRadiusBottom = css<BorderStyleProps>`
     parseBorderRadius,
   )}
 `;
-
+const borderStrokeStyle = css<BorderStyleProps>`
+  ${responsiveStyle("border-style", (props) => props.$borderStyle)};
```

Comment:
There was no logic to apply `$borderStyle` despite the prop being defined.

Reply:
doh